coretex/cli/modules/node.py

```diff
@@ -255,8 +272,47 @@ def _configureInitScript() -> str:
     return str(path)
 
 
+def checkResourceLimitations() -> None:
+    _, ramLimit = docker.getResourceLimits()
+
+    if ramLimit < config_defaults.MINIMUM_RAM_MEMORY:
+        raise RuntimeError(f"Minimum Node RAM requirement ({config_defaults.MINIMUM_RAM_MEMORY}GB) is higher than your current Docker desktop RAM limit ({ramLimit}GB). Please adjust resource limitations in Docker Desktop settings to match Node requirements.")
+
+
+def isConfigurationValid(config: Dict[str, Any]) -> bool:
+    isValid = True
+    cpuLimit, ramLimit = docker.getResourceLimits()
+
+    if not isinstance(config["nodeRam"], int):
+        errorEcho(f"Invalid config \"nodeRam\" field type \"{type(config['nodeRam'])}\". Expected: \"int\"")
+        isValid = False
+
+    if not isinstance(config["cpuCount"], int):
+        errorEcho(f"Invalid config \"cpuCount\" field type \"{type(config['cpuCount'])}\". Expected: \"int\"")
+        isValid = False
+
+    if cpuLimit < config["cpuCount"]:
+        errorEcho(f"Configuration not valid. CPU limit in Docker Desktop ({cpuLimit}) is lower than the configured value ({config['cpuCount']})")
+        isValid = False
+
+    if ramLimit < config["nodeRam"]:
+        errorEcho(f"Configuration not valid. RAM limit in Docker Desktop ({ramLimit}GB) is lower than the configured value ({config['nodeRam']}GB)")
+        isValid = False
+
+    if ramLimit < config_defaults.MINIMUM_RAM_MEMORY:
+        errorEcho(f"Minimum Node RAM requirement ({config_defaults.MINIMUM_RAM_MEMORY}GB) is higher than your current Docker desktop RAM limit ({ramLimit}GB). Please adjust resource limitations in Docker Desktop settings to match Node requirements.")
+        isValid = False
+
+    if config_defaults.MINIMUM_RAM_MEMORY > config["nodeRam"]:
```

Comment:
Place `config` value on the left side of the `>` operator. We are comparing that value against something, not the other way around. Do this for all `if` statements in the function.